docs/tailwind.config.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
         green: colors.green,
         red: colors.red,
         yellow: colors.yellow,
+        "pink-gradient-start": "rgba(255, 30, 86, 1)",
```

Comment:
We might want to unify these in the future and do something like:

```
colors: {
   brand-pink: "...",
   brand-red: "...",
   brand-blue: "...",
}
```

But looks good for now!